Changelog.md

```diff
@@ -0,0 +1,28 @@
+# Changelog
+
+## v0.2.0
+
+Supported Go Versions for this release:
+* Go 1.16
+* Go 1.15
+
+Changes:
+* support for ITE-6 data structures
+* support for ITE-5 signature generation
+* support for specifying a local directory when running inspections
+* switch from golang.org/x/crypto/ed25519 to crypto/ed25519
+
+Thanks to our contributors: Aditya Sirish, Christian Rebischke, cindykimxp, Dan Lorenc, Fredrik Skogman, Lukas Puehringer, Omer Levi Hevroni, Radu M, Santiago Torres, Scott Buckel, seb-bah, Vaibhav Lodha
```

Comment:
```suggestion
Thanks to our contributors: Aditya Sirish, Christian Rebischke, Cindy Kim, Dan Lorenc, Fredrik Skogman, Lukas Puehringer, Omer Levi Hevroni, Radu M, Santiago Torres, Scott Buckel, seb-bah, Vaibhav Lodha
```

Reply:
Not sure who seb-bah is.

Reply:
it's someone who did not configure his git client properly...